uptrain/framework/evalllm.py

```diff
@@ -79,9 +80,9 @@ def evaluate(
                 req_attrs.update([schema.conversation])
 
             if isinstance(m, ParametricEval):
-                ser_checks.append({"check_name": m.__class__.__name__, **m.dict()})
+                ser_checks.append({"check_name": m.__class__.__name__, **m.dict(), "scenario_description": scenario_description})
```

Comment:
Why not add scenario_description to m.dict()? @ashish-1600

Reply:
if isinstance(m, ParametricEval):
         dictn = m.dict()
         dictn.update({"scenario_description": scenario_description})
         ser_checks.append({"check_name": m.__class__.__name__, **dictn)

Reply:
@ashish-1600